jinad/store.py

```diff
@@ -0,0 +1,135 @@
+import uuid
+from typing import List, Dict, Union
+from ruamel.yaml import YAML
+from contextlib import contextmanager
+from jina.peapods.pod import MutablePod
+
+from helper import Flow
+from jina.logging import JinaLogger
+from models.pod import PodModel
+from excepts import FlowYamlParseException, FlowCreationFailed
+
+
+class InMemoryStore:
```

Comment:
Right now, who is imposing that you call the `_session`?

Reply:
I mean, right now `holding` the session is not imposed, which seems a little dangerous right?

Reply:
During the API invocation, this becomes our context manager. So it is invoked in the code itself. I need to look at FastAPI security docs on how to impose this with session tokens.

Reply:
whay I mean, nothing stopes me from writihg `InMemoryFlowStore.create()` without ensuring I have the rights to

Reply:
Maybe use a `Flag` saying that u are `authenticated` or something?